guides/common/modules/con_how-to-configure-inter-server-sync.adoc

```diff
@@ -1,7 +1,7 @@
 [id="how-to-configure-inter-server-sync_{context}"]
-= How to Configure Inter-Server Synchronization
+= How to Configure Inter-Satellite Synchronization
 
-{ProjectName} uses Inter-Server Synchronization (ISS) to synchronize content between two {ProjectServer}s in a multi-server {Project} setup.
+{ProjectName} uses Inter-Satellite Synchronization (ISS) to synchronize content between two {ProjectServer}s in a multi-server {Project} setup.
```

Comment:
Can we define it as an attribute so you can write:
```suggestion
{ProjectName} uses {Project}'s {ISS} (ISS) to synchronize content between two {ProjectServer}s including those that are air-gapped.
```

This is mostly because Satellite is downstream branding and an attribute allows us to use `Inter-Server Synchronization` in upstream. It also means you consistently use the same capitalization.

Reply:
Thanks for this superb idea @ekohl :heart: 

I shall create a new attribute `ISS = Inter-Server Synchronization` to justice upstream and downstream both.

Reply:
Addressed :+1: